sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallConnectionAsync.java

```diff
@@ -945,20 +901,23 @@ public String getCallConnectionId() {
     public Mono<PlayAudioResult> PlayAudioToParticipant(
         String participantId,
         String audioFileUri,
-        String audioFileId,
-        String callbackUri,
-        String operationContext
+        PlayAudioOptions playAudioOptions,
+        final Context context
```

Comment:
The Azure sdk guidance https://azure.github.io/azure-sdk/java_introduction.html said :
✅ DO provide an overload method that takes a com.azure.core.util.Context argument for each service operation **in sync clients only.** 
We have to remove this parameter in any existed api as well.
You can leave the existing apis for later changing or help close this item https://skype.visualstudio.com/SPOOL/_workitems/edit/2577465